app/resources/create.html.template

```diff
@@ -572,11 +571,9 @@
 
         </div>
       </div>
-      </div>
 
-      <div class="column end"></div>
       <div class="end-multi-columns"></div>
```

Comment:
nothing changed when i deleted the <div class="end-multi-columns"></div> 
so i delete all <div class="end-multi-columns"></div> in create.html